### ISSUE TITLE
Accessibility: duplicated elements IDs

### ISSUE THREAD

**Issue by flutterflies** [uclibs/scholar_uc#1685]
### Descriptive summary

No two elements on a page should have the same id attribute.

Two elements on the User Dashboard currently have the same element

```html
<div class="panel panel-default" id="activity">
  <div class="panel-heading dashboard-panel">
    <h3 class="panel-title ">User Activity</h3>
  </div>
  <div class="panel-body">
    <table id="activity" class="table table-striped table-bordered">
      ...
    </table>
  </div>
</div>
```
